src/Coordination/KeeperServer.cpp

```diff
@@ -161,6 +161,24 @@ struct KeeperServer::KeeperRaftServer : public nuraft::raft_server
         reconfigure(new_config);
     }
 
+    uint64_t getFollowersCount()
+    {
+        if (!is_leader())
+        {
+            return 0;
+        }
+        return get_num_voting_members() - 1 - get_not_responding_peers();
+    }
+
+    uint64_t getSyncedFollowerCount()
+    {
+        if (!is_leader())
+        {
+            return 0;
+        }
+        return get_num_voting_members() - 1 - get_num_stale_peers();
```

Comment:
get_num_stale_peers() returns 0 if this node is not a leader, but same as above, we call this function when we are leader. I would leave the manual counting as before.

Reply:
The same as above, this function will be meaningful only when server is leader.
And in deed, before the function is calling, the role of this server will be checked.